features/async-clipboard.yml

```diff
@@ -20,9 +20,8 @@ status:
     safari_ios: "13.4"
 compat_features:
   - api.Clipboard
-  # TODO: include after https://github.com/mdn/browser-compat-data/pull/23593
```

Comment:
I have a feeling we should keep this comments, and others like it. There's some good research in there, which I would hate to lose.